admin/app/controllers/admin/AnalyticsConfidenceController.scala

```diff
@@ -26,7 +28,13 @@ class AnalyticsConfidenceController extends Controller with Logging with AuthLog
         }
       }
 
-      Ok(views.html.lineCharts("PROD", Seq(omnitureGraph, ophanGraph)))
+      val googleGraph = new AwsLineChart("Google confidence", Seq("Time", "%", "avg."), ChartFormat(Colour.`tone-comment-1`, Colour.success)) {
+        override lazy val dataset = ophan.dataset.map{ point =>
```

Comment:
Oups sorry. didn't spot this.
We might want a easier way to define a chart without having to duplicate and copy/paste those few lines.